server/src/main/java/io/unitycatalog/server/persist/utils/RepositoryUtils.java

```diff
@@ -80,4 +80,20 @@ public static CatalogInfoDAO getCatalogByIdOrThrow(Session session, UUID catalog
     }
     return catalogInfoDAO;
   }
+
+  public static Pair<String, String> getCatalogAndSchemaNames(Session session, UUID schemaId) {
+    SchemaInfoDAO schemaInfoDAO =
```

Comment:
Nit:  our checkstyle seems need to make a line as long as possible , so that it reach the 100 chars. 

```java
SchemaInfoDAO schemaInfoDAO = session.get(SchemaInfoDAO.class, schemaId);
    if (schemaInfoDAO == null) {
      throw new BaseException(
          ErrorCode.NOT_FOUND, "Schema not found: " + schemaId);
    }
```

Let me consider how to enforce it.